examples/rabbitmq/rabbitmq_test.go

```diff
@@ -0,0 +1,38 @@
+package rabbitmq
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	amqp "github.com/rabbitmq/amqp091-go"
+)
+
+func TestRabbitMQ(t *testing.T) {
+	ctx := context.Background()
+
+	container, err := startContainer(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Clean up the container after the test is complete
+	t.Cleanup(func() {
+		if err := container.Terminate(ctx); err != nil {
+			t.Fatalf("failed to terminate container: %s", err)
+		}
+	})
+
+	// perform assertions
+
+	amqpConnection, err := amqp.Dial(fmt.Sprintf("amqp://guest:guest@%s", container.endpoint))
```

Comment:
It's great having the amqp connection here. Could you extend the example on how to put/get values on/from the queue? 🙏

Reply:
Ok
I'm just wondering what's the best way to do this?
1. Write and read in one connection
2. Or separate

Reply:
I'm not an expert in RabbitMQ, so I'd suggest adding the most educational example on how to do it.